src/utils/backend/WampClient.js

```diff
@@ -117,30 +117,13 @@ export default class WampClient {
   }
 
   onOpen = ({ sessionId }) => {
-    this.onConnected()
     if (sessionId !== this.id) {
       this.id = sessionId
       log('new session id %s', this.id)
       this.out.emit('set:id', this.id)
     }
   }
 
-  onConnected = () => {
```

Comment:
Why remove the onConnected? Did you check that the `this.out.emit` is not used anywhere e.g. `on("connected")`?

Reply:
@nikgraf you're right, fixed. Also changed random number range from 1-2 to 1-10.